static/app/components/events/interfaces/spans/spanGroupBar.tsx

```diff
@@ -154,8 +163,13 @@ class SpanGroupBar extends React.Component<Props> {
           isSpanGroupToggler
           onClick={event => {
             event.stopPropagation();
-
-            toggleSpanGroup();
+            groupType === GroupType.DESCENDANTS
+              ? toggleSpanGroup()
+              : toggleSiblingSpanGroup &&
+                toggleSiblingSpanGroup(
+                  spanGrouping[0].span.op!,
+                  spanGrouping[0].span.description!
+                );
```

Comment:
Appreciate the insight! Will keep this in mind next time 😄

Reply:
Justy FYI @k-fish , I opted to make the params in `toggleSiblingSpanGroup` either a `string` or `undefined`. This way, the implementation for this function (shown [here](https://github.com/getsentry/sentry/blob/b1403e4850ad8f9dd4ee3e603ae1aaa4ad5c0208/static/app/components/events/interfaces/spans/spanTreeModel.tsx#L703)) will still work even if either `op` or `description` are undefined. This is because `undefined` will get coerced into a string, but I realize this might be a bit hacky? Do you think something like this is acceptable, or should I be checking when parameters are undefined or not?